README.md

```diff
@@ -50,9 +51,9 @@
 
 # Prerelease Program
 
-The Photoshop APIs are made available through the Adobe Prelease program. For the ability to make API calls we invite you to join the program. 
+The Photoshop APIs are made available through the Adobe Prelease program. For the ability to make API calls we invite you to join the program.
```

Comment:
Looks like whitespace change. I would not make whitespace changes at the same time as making content changes. Makes it hard to determine intent of commit.